pandas/tests/frame/methods/test_dropna.py

```diff
@@ -243,3 +243,9 @@ def test_dropna_pos_args_deprecation(self):
             result = df.dropna(1)
         expected = DataFrame({"a": [1, 2, 3]})
         tm.assert_frame_equal(result, expected)
+
+    def test_dropna_retains_RangeIndex_when_nothing_dropped(self):
+        # GH#41965
+        df = DataFrame({"a": range(10)})
+        tm.assert_frame_equal(df, df.dropna(), check_index_type=True)
```

Comment:
can you separate these cases (could also parameterize if you would like) and use

```
result=
tm.assert_frame_equal(result, expected, check_index_type=True)
```

and so on

Reply:
changed.
I have removed the second comparison, i.e. with `dropna(axis="index")` because that's the default param, so essentially same as `drona()`.